.github/workflows/lokalise-download.yml

```diff
@@ -13,8 +13,9 @@ jobs:
         env: 
           VAR_LOKALISE_API_TOKEN: ${{ secrets.LOKALISE_RO_API_TOKEN }}
           VAR_LOKALISE_PROJECT_ID: ${{ secrets.LOKALISE_PROJECT_ID }}
+          VAR_LANGUAGES: zh_CN,cs,da,nl,en_au,en_GB,fr,de,ko_KR,nb,pl,pt_PT,ru_RU,sr,sr_Latn,es,sv_SE,uk_UA
         run: |
-            ./bin/lokalise2 --token ${{ env.VAR_LOKALISE_API_TOKEN }} --project-id ${{ env.VAR_LOKALISE_PROJECT_ID }} file download --format json --export-empty-as skip --export-sort first_added --placeholder-format printf --plural-format array --indentation 4sp --directory-prefix src/translations
+            ./bin/lokalise2 --token ${{ env.VAR_LOKALISE_API_TOKEN }} --project-id ${{ env.VAR_LOKALISE_PROJECT_ID }} file download --filter-langs ${{ env.LANGUAGES }} --format json --export-empty-as skip --export-sort first_added --placeholder-format printf --plural-format array --indentation 4sp --directory-prefix src/translations
```

Comment:
env.LANGUAGES needs to be env.VAR_LANGUAGES other than that everything looks good

Reply:
Fixed